rules/sonic-ztp.mk

```diff
@@ -5,6 +5,7 @@ SONIC_ZTP_VERSION = 1.0.0
 
 SONIC_ZTP = sonic-ztp_$(SONIC_ZTP_VERSION)_all.deb
 $(SONIC_ZTP)_SRC_PATH = $(SRC_PATH)/sonic-ztp
+$(SONIC_ZTP)_WHEEL_DEPENDS +=  $(SWSSSDK_PY3)
```

Comment:
Remove extra space after `+=`

Reply:
@jleveque Request change has been done can you please mark it as complete.